tensorflow/lite/tools/versioning/op_version.cc

```diff
@@ -447,7 +447,27 @@ int GetBuiltinOperatorVersion(const OpSignature& op_sig) {
       }
       return 1;
 
+    case BuiltinOperator_ADD:
+      if (op_sig.input_types.at(0) == TensorType_INT16 &&
+          op_sig.output_types.at(0) == TensorType_INT16) {
+        if (op_sig.options.addsub.pot_scale_int16) {
+          return 4;
+        } else {
+          return 3;
+        }
+      }
+      if (op_sig.input_types.at(0) == TensorType_INT8) {
+        return 2;
+      }
+      return 1;
+
     case BuiltinOperator_SUB:
+      if (op_sig.input_types.at(0) == TensorType_INT16 &&
+          op_sig.output_types.at(0) == TensorType_INT16) {
+        if (op_sig.options.addsub.pot_scale_int16) {
+          return 5;
```

Comment:
This CL got rolled back for an unrelated reason, but shouldn't the "default" case use the existing version? Don't we only need version 5 for the non-pot case?

Reply:
I ask because this broke some internal conversion test cases, as the converter seems to be eagerly setting the version to 5, despite nothing having changed in the model (because pot_scale_int16 is getting set to true by default).

Reply:
Hi @jdduke I set it to 5 here, because INT16 and pot case was not versioned before - maybe, this is wrong. Please let me know if I need to correct this  - Should it be a separate PR ?
Thanks

Reply:
Probably an oversight on our part, as we had some internal tooling using the existing int16 kernels. But even then, this condition seems odd, shouldn't the *new* behavior (non-pot-scale) get the highest op version?

Reply:
Yes, you are right @jdduke. I will correct. Should it be a separate PR ?
I have a task to write versioning tests for ADD/SUB. I will include this change into this new PR. Is it okay ?

Reply:
Nah, that's OK. I'll fix it in the internal CL, just wanted to confirm. Thanks!

Reply:
Hi @jdduke I noticed that these changes are still rolled back. Is there anything I can help with to get it merged ? Thanks